timify-api/app.js

```diff
@@ -72,6 +72,44 @@ export async function searchForTerm(term, page, limitPerPage, filterForGenres) {
 }
 
 
+export async function getPodcastSeries(uuid) {
+  const query = `
+    query getPodcastSeries($uuid: ID!) {
+      getPodcastSeries(uuid: $uuid ) {
+        uuid
+        name
+        datePublished
+        description
+        imageUrl
+        genres
+        seriesType
+        copyright
+        authorName
+        isBlocked
+        episodes{
+          uuid
+          name
+          datePublished
+          description
+          audioUrl
+          fileLength
+          fileType
+          duration
+          seasonNumber
+          episodeNumber
+        }
+      }
+    }
+  `;
+
+  const data = await taddyGraphqlRequest(query, { uuid });
+    if (data.errors) {
+      console.error(data.errors)
+    }
+  return data.data.getPodcastSeries; 
```

Comment:
Why does it have a data field inside data? The name looks a bit confusing. If that's expected then maybe request.data.getPoscastSeries should be better